### ISSUE TITLE
Crash when trimming sectors while dumping to CCD

### ISSUE THREAD

**Issue by SilasLaspada** [aaru-dps/Aaru#276]
### Prerequisites

* [X] Are you running the latest version?
* [X] Can you reproduce the problem in the debug version?

### Check and fill as appropriate:
* [X] I was running DiscImageChef under Windows 7...
* [X] ...in 32-bit

### Description

When dumping a specific CD (it has one data track first and then 9 audio tracks), chef crashes when trimming sectors and only when dumping to CCD, CUE works fine. What's odd is that when dumping to CUE it does trim the sectors consistently, but it ends up saying that no sectors are bad. 
Logs (including media info): https://drive.google.com/open?id=1ziPM8UU-Y0soZ3bnbBFNLA6ijMysaJA8

### Exact command line used:

`DiscImageChef.dll -d m dump e: tomb2.ccd --metadata false -f`

### If command was expected to work with a device fill this out:

* **Manufacturer**: Plextor
* **Model**: PX-760A
* **Bus**:
    * [ ] Parallel ATA
    * [ ] Serial ATA
    * [ ] SCSI (any)
    * [ ] ATAPI (mark above if parallel or serial)
    * [ ] USB
    * [ ] FireWire
    * [ ] PCMCIA
    * [ ] SecureDigital
    * [ ] MultiMediaCard
    
### Output of command execution with debug output enabled
```
Reading sector 164797 of 165951 (4.400 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 164797, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 164828 of 165951 (4.400 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 164828, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 12.6953 ms.
Reading sector 164859 of 165951 (4.400 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 164859, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 164890 of 165951 (4.400 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 164890, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 164921 of 165951 (4.400 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 164921, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 12.6953 ms.
Reading sector 164952 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 164952, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 164983 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 164983, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 11.7187 ms.
Reading sector 165014 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165014, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 165045 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165045, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 165076 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165076, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 12.6954 ms.
Reading sector 165107 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165107, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 165138 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165138, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6485 ms.
Reading sector 165169 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165169, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 165200 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165200, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 165231 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165231, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 165262 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165262, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 8.7891 ms.
Reading sector 165293 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165293, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 165324 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165324, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6718 ms.
Reading sector 165355 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165355, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 165386 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165386, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 165417 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165417, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 4.8829 ms.
Reading sector 165448 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165448, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 165479 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165479, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 12.6953 ms.
Reading sector 165510 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165510, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 165541 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165541, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 165572 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165572, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 165603 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165603, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6485 ms.
Reading sector 165634 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165634, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 12.6953 ms.
Reading sector 165665 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165665, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 165696 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165696, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 14.6484 ms.
Reading sector 165727 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165727, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 165758 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165758, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 165789 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165789, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 13.6719 ms.
Reading sector 165820 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165820, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 12.6953 ms.
Reading sector 165851 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165851, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 2.9297 ms.
Reading sector 165882 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165882, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 2.9297 ms.
Reading sector 165913 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165913, Block Size: 2448, Transfer Length: 32, Expected Sector Type: AllTypes
, DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Dat
a: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0)
took 3.9062 ms.
Reading sector 165944 of 165951 (4.421 MiB/sec.)DEBUG (SCSI Device): READ CD (LB
A: 165944, Block Size: 2448, Transfer Length: 8, Expected Sector Type: AllTypes,
 DAP: False, Relative Address: False, Sync: True, Headers: AllHeaders, User Data
: True, ECC/EDC: True, C2: None, Subchannel: Raw, Sense: False, Last Error: 0) t
ook 1.9531 ms.

Dump finished in 115.1367187 seconds.
Average dump speed 3940.966 KiB/sec.
Average write speed 156793.149 KiB/sec.
Trimming bad sectors
Trimming sector 87200DEBUG (SCSI Device): READ CD (LBA: 87200, Block Size: 2448,
 Transfer Length: 1, Expected Sector Type: AllTypes, DAP: False, Relative Addres
s: False, Sync: True, Headers: AllHeaders, User Data: True, ECC/EDC: True, C2: N
one, Subchannel: Raw, Sense: False, Last Error: 0) took 115.2344 ms.
Trimming sector 87201DEBUG (SCSI Device): READ CD (LBA: 87201, Block Size: 2448,
 Transfer Length: 1, Expected Sector Type: AllTypes, DAP: False, Relative Addres
s: False, Sync: True, Headers: AllHeaders, User Data: True, ECC/EDC: True, C2: N
one, Subchannel: Raw, Sense: False, Last Error: 0) took 0 ms.
Trimming sector 87202DEBUG (SCSI Device): READ CD (LBA: 87202, Block Size: 2448,
 Transfer Length: 1, Expected Sector Type: AllTypes, DAP: False, Relative Addres
s: False, Sync: True, Headers: AllHeaders, User Data: True, ECC/EDC: True, C2: N
one, Subchannel: Raw, Sense: False, Last Error: 0) took 0.9766 ms.
Trimming sector 87203DEBUG (SCSI Device): READ CD (LBA: 87203, Block Size: 2448,
 Transfer Length: 1, Expected Sector Type: AllTypes, DAP: False, Relative Addres
s: False, Sync: True, Headers: AllHeaders, User Data: True, ECC/EDC: True, C2: N
one, Subchannel: Raw, Sense: False, Last Error: 0) took 0.9765 ms.
Trimming sector 87204DEBUG (SCSI Device): READ CD (LBA: 87204, Block Size: 2448,
 Transfer Length: 1, Expected Sector Type: AllTypes, DAP: False, Relative Addres
s: False, Sync: True, Headers: AllHeaders, User Data: True, ECC/EDC: True, C2: N
one, Subchannel: Raw, Sense: False, Last Error: 0) took 0.9765 ms.
Trimming sector 87205DEBUG (SCSI Device): READ CD (LBA: 87205, Block Size: 2448,
 Transfer Length: 1, Expected Sector Type: AllTypes, DAP: False, Relative Addres
s: False, Sync: True, Headers: AllHeaders, User Data: True, ECC/EDC: True, C2: N
one, Subchannel: Raw, Sense: False, Last Error: 0) took 174.8047 ms.
Trimming sector 87206DEBUG (SCSI Device): READ CD (LBA: 87205, Block Size: 2448,
 Transfer Length: 2, Expected Sector Type: AllTypes, DAP: False, Relative Addres
s: False, Sync: True, Headers: AllHeaders, User Data: True, ECC/EDC: True, C2: N
one, Subchannel: Raw, Sense: False, Last Error: 0) took 227.539 ms.
Unhandled exception: System.Reflection.TargetInvocationException: Exception has
been thrown by the target of an invocation. ---> System.ArgumentException: Sourc
e array was not long enough. Check the source index, length, and the array's low
er bounds.
Parameter name: sourceArray
   at System.Array.Copy(Array sourceArray, Int32 sourceIndex, Array destinationA
rray, Int32 destinationIndex, Int32 length, Boolean reliable)
   at System.Array.Copy(Array sourceArray, Int64 sourceIndex, Array destinationA
rray, Int64 destinationIndex, Int64 length)
   at DiscImageChef.Core.Devices.Dumping.Dump.TrimCdUserData(ExtentsULong audioE
xtents, UInt32 blockSize, DumpHardwareType currentTry, ExtentsULong extents, Boo
lean newTrim, Int32 offsetBytes, Boolean read6, Boolean read10, Boolean read12,
Boolean read16, Boolean readcd, Int32 sectorsForOffset, UInt32 subSize, MmcSubch
annel supportedSubchannel, Boolean supportsLongSectors, Double& totalDuration) i
n C:\Users\silas\Desktop\GIT\DICF\DiscImageChef\DiscImageChef.Core\Devices\Dumpi
ng\CompactDisc\Trim.cs:line 159
   at DiscImageChef.Core.Devices.Dumping.Dump.CompactDisc() in C:\Users\silas\De
sktop\GIT\DICF\DiscImageChef\DiscImageChef.Core\Devices\Dumping\CompactDisc\Dump
.cs:line 1007
   at DiscImageChef.Core.Devices.Dumping.Dump.Mmc() in C:\Users\silas\Desktop\GI
T\DICF\DiscImageChef\DiscImageChef.Core\Devices\Dumping\MMC.cs:line 198
   at DiscImageChef.Core.Devices.Dumping.Dump.Scsi() in C:\Users\silas\Desktop\G
IT\DICF\DiscImageChef\DiscImageChef.Core\Devices\Dumping\SCSI.cs:line 237
   at DiscImageChef.Core.Devices.Dumping.Dump.Start() in C:\Users\silas\Desktop\
GIT\DICF\DiscImageChef\DiscImageChef.Core\Devices\Dumping\Dump.cs:line 160
   at DiscImageChef.Commands.Media.DumpMediaCommand.Invoke(Boolean debug, Boolea
n verbose, String cicmXml, String devicePath, Boolean resume, String encoding, B
oolean firstPregap, Boolean fixOffset, Boolean force, Boolean metadata, Boolean
trim, String outputPath, String options, Boolean persistent, UInt16 retryPasses,
 UInt32 skip, Byte speed, Boolean stopOnError, String format, String subchannel)
 in C:\Users\silas\Desktop\GIT\DICF\DiscImageChef\DiscImageChef\Commands\Media\D
ump.cs:line 431
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments,
 Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invoke
Attr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.CommandLine.Invocation.ModelBindingCommandHandler.InvokeAsync(Invoc
ationContext context) in /_/src/System.CommandLine/Invocation/ModelBindingComman
dHandler.cs:line 62
   at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<Bu
ildInvocationChain>b__0>d.MoveNext() in /_/src/System.CommandLine/Invocation/Inv
ocationPipeline.cs:line 56
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<UseParseErrorRepo
rting>b__22_0>d.MoveNext() in /_/src/System.CommandLine/Invocation/InvocationExt
ensions.cs:line 367
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c__DisplayClass8_0.<<
UseTypoCorrections>b__0>d.MoveNext() in /_/src/System.CommandLine/Invocation/Inv
ocationExtensions.cs:line 225
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<UseSuggestDirecti
ve>b__7_0>d.MoveNext() in /_/src/System.CommandLine/Invocation/InvocationExtensi
ons.cs:line 206
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<UseParseDirective
>b__6_0>d.MoveNext() in /_/src/System.CommandLine/Invocation/InvocationExtension
s.cs:line 177
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<UseHelp>b__20_0>d
.MoveNext() in /_/src/System.CommandLine/Invocation/InvocationExtensions.cs:line
 335
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass4
_0.<<UseVersionOption>b__0>d.MoveNext() in /_/src/System.CommandLine/Builder/Com
mandLineBuilderExtensions.cs:line 77
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c.<<RegisterWithDotne
tSuggest>b__23_0>d.MoveNext() in /_/src/System.CommandLine/Invocation/Invocation
Extensions.cs:line 406
--- End of stack trace from previous location where exception was thrown ---
   at System.CommandLine.Invocation.InvocationExtensions.<>c__DisplayClass5_0.<<
UseExceptionHandler>b__0>d.MoveNext() in /_/src/System.CommandLine/Invocation/In
vocationExtensions.cs:line 142
```

**Comment by SilasLaspada** [comment]
Seems to occur when dumping to DICF too.

**Comment by claunia** [comment]
Fixed in eba2452afc85b0f3592c91545a12ed691f25a9c4.
